### PR TITLE
Feature/add idea to local sorage

### PR DESCRIPTION
**Pull request by justincanthony**
## Is this a fix or a feature?
  *feature
## What is the change?
  *we can save cards to localStorage and update the favorited status in local storage
## What does it fix?
  *n/a
## Where should the reviewer start?
  *with idea.js and main,js
## How should this be tested?
  * open index.html

### CODE REVIEW

**Review by jfargnoli01 — APPROVED** [review]
Great job, Justin!

File: main.js [hunk 1]
```diff
@@ -3,67 +3,30 @@ var saveButton = document.querySelector('.save-btn');
...
-var ideaTitle = document.querySelector('.form-title'); // title text input
-var ideaBody = document.querySelector('.form-body'); // body text input
+var ideaTitle = document.querySelector('.form-title');
+var ideaBody = document.querySelector('.form-body');
 var searchText = document.querySelector('.search-field');
 var renderedIdeas = document.querySelector('.ideabox');
-var savedIdeas = [];
 
+var savedIdeas = [];
+console.log(savedIdeas);
```
Comment (dreardon1021) [inline main.js]: Remove this console.log

PR's should not contain them. I have this issue myself. I find best way to solve is to just keyword search in your text editor before you push

File: main.js [hunk 2]
```diff
@@ -74,8 +37,10 @@ function instantiateIdeaClass(event) {
   } else {
     saveButton.disabled = false;
     var newIdeaObject = new Idea(ideaTitle.value, ideaBody.value);
+    console.log(newIdeaObject);
```
Comment (dreardon1021) [inline main.js]: remove console here

**Review by dreardon1021 — APPROVED** [review]
Looks good. Just remove some console.logs and youre set

**Review by Matt-Roden — APPROVED** [review]
Looks good! 🥷

File: main.js (hunk 1 above)
Comment (justincanthony) [reply main.js]: Thank you for the feedback on this one!

**Pull request merged**